sdks/go/pkg/beam/core/runtime/harness/harness.go

```diff
@@ -243,7 +243,7 @@ func fail(id, format string, args ...interface{}) *fnpb.InstructionResponse {
 func dial(ctx context.Context, endpoint string, timeout time.Duration) (*grpc.ClientConn, error) {
 	log.Infof(ctx, "Connecting via grpc @ %s ...", endpoint)
 
-	opts := []grpc.DialOption{grpc.WithInsecure(), grpc.WithBlock()}
+	opts := []grpc.DialOption{grpc.WithInsecure(), grpc.WithBlock(), grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(50 << 20))}
```

Comment:
Please make this change in the grpcx helper as well (which this code is incidentally replaced with in #4265).

Reply:
Done. Thanks for taking care of the timeout change in the new code!

Reply:
Something went sideways with my Git. I'm going to abandon this and try again.